### DISCUSSION TITLE
Can this be used for supersonic flows?

### DISCUSSION THREAD

**Discussion by poulet67** [ProjectPhysX/FluidX3D#98] [Q&A] [answered]
Hi, please forgive my ignorance I am not an expert in the subject matter and can't figure out a firm answer reading the docs.

Is this program only suitable for simulating subsonic flows? 

Thanks

**Comment by ibonito1** [comment, marked as answer]
No. Strictly speaking, it's not even suitable to subsonic flows, but only to incompressible (or very mildly compressible flows with M < 0.3).  

https://github.com/ProjectPhysX/FluidX3D/issues/97#issuecomment-1660509722

**Reply by poulet67** [reply]
I suspected as much. Thanks!
